src/main/io/vtx_tramp.h

```diff
@@ -45,6 +45,7 @@ extern uint16_t trampConfiguredPower; // Configured transmitting power
 extern int16_t trampTemperature;
 
 bool vtxTrampInit(void);
+void vtxTrampSetPowerByIndex(uint8_t index);
```

Comment:
The `trampSetRFPower()` should be removed from the header file and made static in `vtx_tramp.c`, as it is not intended to be called externally.

Reply:
Done.